docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -24,18 +24,19 @@ deploymentConfig {
 
 ## Kubernetes native Lite engine configuration
 
-Please note, that K8s Deployment Manager has to be run with properly configured K8s access. If you install the Designer
-in K8s cluster (e.g. via Helm chart) this comes out of the box. If you want to run the Designer outside the cluster, you
-have to configure `.kube/config` properly.
+Please check high level [Lite engine description](https://nussknacker.io/documentation/about/engines/LiteArchitecture/#scenario-deployment) before proceeding to configuration details. 
+
+Please note, that K8s Deployment Manager has to be run with properly configured K8s access. If you install the Designer in K8s cluster (e.g. via Helm chart) this comes out of the box. If you want to run the Designer outside the cluster, you have to configure `.kube/config` properly.
 
 Except the `servicePort` configuration option, all remaining configuration options apply to
-both `streaming` and `request-response` processing modes.
+both `streaming` and `request-response` processing modes. 
+
+The table below contains configuration options for the Lite engine. If you install Designer with Helm, you can use Helm values override mechanism to supply your own values for these [options](https://artifacthub.io/packages/helm/touk/nussknacker#configuration-in-values-yaml). As the the result of the Helm template rendering "classic" Nussknacker configuration file will be generated.
```

Comment:
The only non editorial change

Reply:
I would like to point out that names of these parameters in our helm chart's values can be different than presented in the table.